edn/src/symbols.rs

```diff
@@ -277,3 +277,29 @@ impl Display for NamespacedKeyword {
         write!(f, ":{}/{}", self.namespace, self.name)
     }
 }
+
+macro_rules! ns_keyword {
+    ($name: expr) => {{
+        let splits = [':', '/'];
+        let mut i = $name.split(&splits[..]);
```

Comment:
I realized a while back that this isn't quite right -- it will accept the splits in the wrong order, like `/bad:keyword`.  File a follow-up to do this in two stages?